integration_tests/benchmarks/index.js

```diff
@@ -28,32 +28,65 @@ async function runBenchmark(artifactsDir, modelName, config) {
   // for losses.
 
   const batchSize = benchmarkData.batch_size;
-  const xs = tfc.randomUniform([batchSize].concat(benchmarkData.input_shape));
-  const ys = tfc.randomUniform([batchSize].concat(benchmarkData.target_shape));
-  model.compile({
-    optimizer: benchmarkData.optimizer,
-    loss: lossMap[benchmarkData.loss],
-  });
+  let xs;
```

Comment:
Can we pull the shape inference out into a helper function?  (Yes I doubt it would be reused, but this runBenchmark function is getting long/harder to follow.